va_explorer/users/forms.py

```diff
@@ -247,6 +294,28 @@ def save(self, commit=True):
             user.location_restrictions.set(location_restrictions)
             user.groups.set([group])
 
+            # TODO: Update this if we are supporting more than one username; for now, allow only one
```

Comment:
Do you think it would be worthwhile to have a `set_va_username(*usernames)` method on User model that encapsulates all the logic for adding/removing usernames? This would allow us to use/set the username outside of a form easily if we need to do so.

Reply:
@timster Yes! I think that would be great! I will implement your suggestion.